### PR TITLE
Basic Kotlin project structure

### PR DESCRIPTION
**Pull request by quietbits**
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
```
Comment (lijamie98) [inline .gitignore:4]: `.idea/` will ignore all files generated by IntelliJ or Android Studio.

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    testImplementation(kotlin("test"))
+    implementation("com.github.stellar:java-stellar-sdk:0.34.2")
+}
+
+tasks.test {
+    useJUnitPlatform()
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions.jvmTarget = "1.8"
```
Comment (lijamie98) [inline build.gradle.kts:26]: 1.8 is a little low. It should be safe to use 1.11 which is more modern and is one of the most popular version for backward compatibility.

File: build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```
Comment (lijamie98) [inline build.gradle.kts:1]: I am not sure if we would want to use Kotlin DSL for Gradle. Although the anchor platform uses Kotlin DSL, most public Gradle examples are written in Groovy.

File: src/main/kotlin/org/stellar/kotlinwalletsdk/Wallet.kt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package org.stellar.kotlinwalletsdk
```
Comment (lijamie98) [inline src/main/kotlin/org/stellar/kotlinwalletsdk/Wallet.kt:1]: In common practice, package names do not use `kotlin`. 

for example: `org.stellar.walletsdk` or `org.stellar.wallet.sdk`

**Comment by lijamie98** [comment]
`.idea` folder should be in `.gitignore`

File: build.gradle.kts (hunk 3 above)
Comment (quietbits) [reply build.gradle.kts:1]: Sure, we can use Groovy for Gradle. I saw Kotlin option and just went all in on Kotlin. :sweat:

File: src/main/kotlin/org/stellar/kotlinwalletsdk/Wallet.kt (hunk 4 above)
Comment (quietbits) [reply src/main/kotlin/org/stellar/kotlinwalletsdk/Wallet.kt:1]: If we later create Wallet SDK in Java, how would we name it? I added Kotlin there for that reason, but we can remove it.

**Comment by quietbits** [comment]
@lijamie98, thanks for your review! I included `.idea` to "force" the same formatting for Kotlin files. In JavaScript, there is Prettier that does that. Is there a way to do the same in Kotlin/Java? Or maybe it's not a convention used in Kotlin/Java?

**Review by lijamie98 — APPROVED** [review]
LGTM

**Comment by lijamie98** [comment]
> @lijamie98, thanks for your review! I included `.idea` to "force" the same formatting for Kotlin files. In JavaScript, there is Prettier that does that. Is there a way to do the same in Kotlin/Java? Or maybe it's not a convention used in Kotlin/Java?

Yes, we can enable Google Jave Style and ktfmt plug-in for Kotlin in Gradle. I can help with that. 

You may enable both plugins in IntelliJ that will help with automatic formatting

**Pull request merged**